renovate.json5

```diff
@@ -7,25 +7,23 @@
     {
       // Those cannot be upgraded to a major version until we drop support for Node 10
       packageNames: [
-        'path-type',
```

Comment:
I sorted these by mistake 👼

Reply:
Going to move those to https://github.com/netlify/renovate-config/blob/1cb5cafdb879e7285991f3e23f94c3888c73f1a4/default.json#L12